doc/changelog.rst

```diff
@@ -14,20 +14,22 @@ Description
 
 - This version adds new functionality in the form of support for Unix Domain Sockets.
 - Fortran client can now be optionally built with the rest of the library
-- Initial support for dataset conversions, specifically Xarray. 
+- Initial support for dataset conversions, specifically Xarray.
 
 Detailed Notes
 
 - Implemented support for Unix Domain Sockets, including refactorization of server address code, test cases, and check-in tests. (PR252_)
-- A new make target `make lib-with-fortran` now compiles the Fortran client and dataset into its own library which applications can link against (245_)
+- A new make target `make lib-with-fortran` now compiles the Fortran client and dataset into its own library which applications can link against (PR245_)
```

Comment:
These should be in reverse chronological order to be consistent with earlier changelogs